stage1_fly/enter/main.go

```diff
@@ -15,7 +15,6 @@
 package main
 
 import (
-	"log"
```

Comment:
it didn't. I filed #2031 so this can get caught by the tests. Also, building all flavors locally now, too. ;)